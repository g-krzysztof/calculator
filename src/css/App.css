.App {
  background-color: #ccc;
  width: 301px;
  height: 401px;
  border: 1px solid #aaa;
  margin: 100px auto 0 auto;
}

.buttons-wrapper {
  height: 319px;
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  -ms-flex-wrap: wrap;
      flex-wrap: wrap;
  -webkit-box-pack: justify;
      -ms-flex-pack: justify;
          justify-content: space-between;
  -ms-flex-line-pack: justify;
      align-content: space-between;
}
/*# sourceMappingURL=App.css.map */