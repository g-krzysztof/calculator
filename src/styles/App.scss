.App {
    background-color: #ccc;
    width: 451px;
    height: 321px;
    border: 1px solid #aaa;
    margin: 100px auto 0 auto;
}

.buttons-wrapper {
    height: 239px;
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
    // align-items: center;
    align-content: space-between;
}