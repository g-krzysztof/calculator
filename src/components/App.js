import React from 'react';
import '../css/App.css';
import Display from './Display';
import Button from './Button';

class App extends React.Component {
  state = {
    display: "42509",
  }
  render() {
    return (
      <div className="App">
        <Display display={this.state.display} />
        <div className="buttons-wrapper">
          <Button name="1" />
          <Button name="2" />
          <Button name="3" />
        </div>
      </div>
    );
  }
}

export default App;