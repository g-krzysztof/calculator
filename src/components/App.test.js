import React from 'react';
// import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   expect(div.className).toBe("App");
//   ReactDOM.unmountComponentAtNode(div);
// });

it('renders a App snapshot', () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});