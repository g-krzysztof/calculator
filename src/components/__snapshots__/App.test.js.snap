// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a App snapshot 1`] = `
<div
  className="App"
>
  <div
    className="Display"
  >
    42509
  </div>
  <div
    className="buttons-wrapper"
  >
    <div
      className="Button"
    >
      <button>
        1
      </button>
    </div>
    <div
      className="Button"
    >
      <button>
        2
      </button>
    </div>
    <div
      className="Button"
    >
      <button>
        3
      </button>
    </div>
  </div>
</div>
`;
